jsons/parameters.json

```diff
@@ -593,7 +594,11 @@
                     true
                 ],
                 [
-                    "mascot_x_x_x",
+                    "mascot_2_5_1",
+                    true
+                ],
+                [
+                    "mascot_2_6_2",
```

Comment:
These changes, should also be included into ursgal_style_1 value translation, otherwise they won't be picked up.

Also not sure if then mascot_x_x_x should be changed to 2_5_1 and 2_6_2 in all translations?